apps/www/pages/r/[id].tsx

```diff
@@ -21,7 +21,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     props: {
       id,
       isUser: session?.user ? true : false,
-      isSponsor: session?.isSponsor,
+      isSponsor: session?.isSponsor ?? false,
```

Comment:
When running locally isSponsor was undefined but I'm not sure why. Added a fallback to false